src/routes/about/+page.js

```diff
@@ -0,0 +1,352 @@
+import { tooltip } from 'svooltip';
+
+const dataSrc = `https://api.github.com/repos/spyder-ide/spyder/contributors?per_page=100`;
+
+const currentContributors = [
+  {
+    id: 365293,
+    name: "Carlos Córdoba",
+    role: "Lead mantainer",
+    tooltip: [
+      {
+        list: [
+           "Maintainer since 2013"
+        ]
+      },
+    ]
+  },
+  {
+    id: 16781833,
+    name: "Daniel Althviz",
+    role: "Co-mantainer",
+    tooltip: [
+      {
+        list: [
+          "Release manager",
+          "Maintainer of QtPy and QtAwesome",
+          "Manager of NumFOCUS small development grants"
+        ]
+      }
+    ]
+  },
+  {
+    id: 17051931,
+    name: "C.A.M. Gerlach",
+    role: "Core developer",
+    avatar_url: "/assets/authors/camgerlach/pic.webp",
+    tooltip: [
+      {
+        list: [
+          "Docs maintainer",
+          "Technical writer"
```

Comment:
```suggestion
          "Technical, grant and blog writer",
          "Outreach lead"
```

Add my outreach role and expand the tech writer a bit